### PR TITLE
Update preparation.md

### PR DESCRIPTION
**Pull request by klenze**
spack needs flock capable fs, or it will hang indefinitely.

### CODE REVIEW

File: docs/preparation.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 
 ### I.2. Clone the repo
+
+If you are running on NFS, please take note the [Filesystem requirements](https://spack.readthedocs.io/en/latest/basic_usage.html#filesystem-requirements) of spack on the ``flock`` requirements of spack and possible workarounds.
+
 ```
 git clone -b dev https://github.com/FairRootGroup/FairSoft
 cd FairSoft
 ```
-
```
Comment (ChristianTackeGSI) [inline docs/preparation.md:10]: Please leave the seperation blank line. IMHO it improves readability.

**Review by ChristianTackeGSI — COMMENTED** [review]
Thanks for looking at our spack docs!

> Update preparation.md 

* Can you be a bit more precise in the main subject of the commit  message?
* Can you please prefix this with `[spack]`?

Maybe something like "\[spack\] Add Note About Filesystem Requirements"?

**Comment by klenze** [comment]
Sorry, this was an automatically generated PR from the github website. I will try to make a better PR manually.

**Comment by ChristianTackeGSI** [comment]
JFI: You could have used force push to continue with this PR. Anyway: Merged the other one. Thanks!

**Pull request closed without merging**